src/utils.ts

```diff
@@ -92,3 +56,23 @@ export function getLatestPrereleaseTag(
     .filter((tag) => prerelease(tag.name))
     .find((tag) => tag.name.match(identifier));
 }
+
+export function mapCustomReleaseRules(
+  customReleaseTypes: string
+) {
+  return customReleaseTypes
+    .split(',')
+    .map(part => {
+      const custom = part.split(':');
+      if (custom.length !== 2) {
+        core.warning(`${part} is not a valid custom release definition.`);
+        return null;
+      }
+      const [keyword, release] = custom;
+      return {
+        type: keyword,
+        release
+      };
+    })
+    .filter(customRelease => DEFAULT_RELEASE_TYPES.includes(customRelease?.release));
```

Comment:
let's show some warning if the release isn't a known release type

Reply:
Good idea! Added.